src/main/java/seedu/address/model/item/Item.java

```diff
@@ -0,0 +1,52 @@
+package seedu.address.model.item;
+
+import static seedu.address.model.util.ItemUtil.APPLICATION_ALIAS;
+import static seedu.address.model.util.ItemUtil.COMPANY_ALIAS;
+import static seedu.address.model.util.ItemUtil.INTERNSHIP_ALIAS;
+import static seedu.address.model.util.ItemUtil.PROFILE_ALIAS;
+
+import java.util.LinkedHashMap;
+
+/**
+ * Represents an Item in the InternHunter application.
+ * There are 4 types of items, InternshipItem, InternshipApplication, CompanyItem, ProfileItem.
```

Comment:
To stick to a standardization, shall we also rename the `InternshipApplication` class to `InternshipApplicationItem`?

Reply:
Yup made the change